draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -183,9 +180,12 @@ hostnames by executing steps 3 to 6 and prepopulate an ICE agent accordingly.
 By doing so, only step 7 of the above procedure will be executed at the time
 of gathering candidates.
 
-An implementation may also detect that mDNS is not supported by the available
-network interfaces. The ICE agent may skip steps 3 and 4 and directly decide to
-not expose the host candidate.
+A web page SHOULD NOT be able to learn any user-specific information from the
```

Comment:
Didn't totally understand the IPHandling reference here. Might also be able to phrase this more simply, e.g., 

> In order to prevent web applications from using this mechanism to query for mDNS support in the local network, the ICE agent SHOULD still provide mDNS candidates even if the local network does not support mDNS or mDNS registration fails.

Reply:
I took the proposed sentences and added references to steps 5 and 6.